com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/ExtenderUtil.java

```diff
@@ -504,6 +527,8 @@ private static IResource getPropertyResource(Project project, BobProjectProperti
             sources.add( new FSExtenderResource( resource ) );
             sources.addAll( listFilesRecursive( project, extension + "/include/" ) );
             sources.addAll( listFilesRecursive( project, extension + "/src/") );
+            sources.addAll( listFilesRecursive( project, extension + "/commonsrc/") );
+            sources.addAll( listFilesRecursive( project, extension + "/pluginsrc/") );
```

Comment:
Separate from the "src/" folder, since we don't want to include the plugin java files in the Android classes.dex file.